cmd/bcpd/app/tx.go

```diff
@@ -52,6 +54,10 @@ func (tx *Tx) GetMsg() (weave.Msg, error) {
 		return t.ReturnEscrowMsg, nil
 	case *Tx_UpdateEscrowMsg:
 		return t.UpdateEscrowMsg, nil
+	case *Tx_CreateContractMsg:
```

Comment:
This one bit me a few times in testing... but the error always tells me. Good to add both